android/app/src/main/java/org/uwaterloo/subletr/pages/createlisting/CreateListingPageViewModel.kt

```diff
@@ -120,16 +123,15 @@ class CreateListingPageViewModel @Inject constructor(
 							description = it.description,
 							residenceType = ResidenceType.house,
 							imgIds = imgIds,
-							roomsAvailable = it.numBedrooms,
-// TODO: ADDING LOCATION
-							latitude = 0f,
-							longitude = 0f,
-//							TODO: Update UI to allow the inputs of these newly added values
+							latitude = location!!.latitude.toFloat(),
+							longitude = location.longitude.toFloat(),
+							// TODO: Update UI to allow the inputs of these newly added values
 							bathroomsAvailable = 0,
 							bathroomsEnsuite = 0,
 							bathroomsTotal = 0,
+							roomsAvailable = 0,
 							roomsTotal = 0,
-							gender = "Female",
+							gender = Resources.getSystem().getString(Gender.OTHER.stringId),
```

Comment:
I assume that we'll fill this later so I think the string literal as a placeholder is fine.